Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/PlayerMovedMessage.java

```diff
@@ -1,45 +1,44 @@
-package nl.soccar.gameserver.message;
-
-import nl.soccar.gameserver.message.handler.PlayerMovedMessageHandler;
-import nl.soccar.library.enumeration.HandbrakeAction;
-import nl.soccar.library.enumeration.SteerAction;
-import nl.soccar.library.enumeration.ThrottleAction;
-import nl.soccar.socnet.message.Message;
-import nl.soccar.socnet.message.MessageConstants;
-import nl.soccar.socnet.message.MessageEvent;
-
-/**
- *
- * @author PTS34A
- */
-@MessageEvent(id = MessageConstants.PLAYER_MOVED_MESSAGE_ID, handler = PlayerMovedMessageHandler.class)
-public final class PlayerMovedMessage extends Message {
-
-    private final SteerAction steerAction;
-    private final HandbrakeAction handbrakeAction;
-    private final ThrottleAction throttleAction;
-
-    public PlayerMovedMessage(SteerAction steerAction, HandbrakeAction handbrakeAction, ThrottleAction throttleAction) {
-        this.steerAction = steerAction;
-        this.handbrakeAction = handbrakeAction;
-        this.throttleAction = throttleAction;
-    }
-
-    public SteerAction getSteerAction() {
-        return steerAction;
-    }
-
-    public HandbrakeAction getHandbrakAction() {
-        return handbrakeAction;
-    }
-
-    public ThrottleAction getThrottleAction() {
-        return throttleAction;
-    }
-
-    @Override
-    public int getId() {
-        return MessageConstants.PLAYER_MOVED_MESSAGE_ID;
-    }
-
-}
+package nl.soccar.gameserver.controller.socnet.message;
+
+import nl.soccar.gameserver.controller.socnet.message.handler.PlayerMovedMessageHandler;
+import nl.soccar.library.enumeration.HandbrakeAction;
+import nl.soccar.library.enumeration.SteerAction;
+import nl.soccar.library.enumeration.ThrottleAction;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+
+/**
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.PLAYER_MOVED_MESSAGE_ID, handler = PlayerMovedMessageHandler.class)
+public final class PlayerMovedMessage extends Message {
+
+    private final SteerAction steerAction;
+    private final HandbrakeAction handbrakeAction;
+    private final ThrottleAction throttleAction;
+
+    public PlayerMovedMessage(SteerAction steerAction, HandbrakeAction handbrakeAction, ThrottleAction throttleAction) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)